src/test/java/com/desiato/puresynth/models/UserControllerTest.java

```diff
@@ -0,0 +1,52 @@
+package com.desiato.puresynth.models;
+
+import com.desiato.puresynth.configurations.SecurityConfig;
+import com.desiato.puresynth.repositories.UserRepository;
+import jakarta.transaction.Transactional;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.web.servlet.AutoConfigureMockMvc;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.context.annotation.Import;
+import org.springframework.http.MediaType;
+import org.springframework.test.web.servlet.MockMvc;
+
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.*;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.*;
+
+@SpringBootTest
+@AutoConfigureMockMvc
+@Import(com.desiato.puresynth.configurations.SecurityConfig.class)
+public class UserControllerTest {
+
+    @Autowired
+    private MockMvc mockMvc;
+
+    @Autowired
+    UserRepository userRepository;
+
+    @BeforeEach
+    @Transactional
```

Comment:
Why do you need this?

Reply:
I thought I needed it, but actually the test works without it.